### PR TITLE
documentation: embed eCHO episodes in Cilium docs

### PR DESCRIPTION
**Pull request by hacktivist123**
Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [ ] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] If your commit description contains a `Fixes: <commit-id>` tag, then
      please add the commit author[s] as reviewer[s] to this issue.
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [ ] Provide a title or release-note blurb suitable for the release notes.
- [ ] Are you a user of Cilium? Please add yourself to the [Users doc](https://github.com/cilium/cilium/blob/main/USERS.md)
- [ ] Thanks for contributing!

This PR embeds eCHO episodes in parts of the documentation that matches them in order to act as support documentation

### CODE REVIEW

File: Documentation/installation/cni-chaining-azure-cni.rst [hunk 1]
```diff
@@ -21,6 +21,13 @@ Azure CNI (Legacy)
 
 .. include:: cni-chaining-limitations.rst
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like a video explanation of the Azure CNI Powered by Cilium, check out `eCHO episode 70: Azure CNI Powered by Cilium <https://www.youtube.com/live/8it8Hm2F_GM?si=E_z57MLGJZeZJSyN>`__.
+
+
+
```
Comment (qmonnet) [inline Documentation/installation/cni-chaining-azure-cni.rst]:
```suggestion
```

File: Documentation/installation/k8s-install-helm.rst [hunk 2]
```diff
@@ -85,21 +85,19 @@ Install Cilium
 
        .. include:: ../installation/requirements-aks.rst
 
+       .. note::
+            Installing Cilium via helm is supported only for AKS BYOCNI cluster and
+            not for Azure CNI Powered by Cilium clusters.
+
```
Comment (qmonnet) [inline Documentation/installation/k8s-install-helm.rst]: This is a separate change, please move it to a separate commit.

File: Documentation/installation/k8s-install-helm.rst [hunk 3]
```diff
@@ -85,21 +85,19 @@ Install Cilium
...
        **Install Cilium:**
 
        Deploy Cilium release via Helm:
 
        .. parsed-literal::
-
```
Comment (qmonnet) [inline Documentation/installation/k8s-install-helm.rst]: No need to remove this line, for my part I prefer keeping a blank line to separate the directive name from its content.

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst [hunk 4]
```diff
@@ -15,6 +15,11 @@ from outside the cluster for environments that support BGP. Because BGP
 Control Plane does not program the :ref:`datapath <ebpf_datapath>`, do not use it to establish
 reachability within the cluster.
 
+.. admonition:: Video
+  :class: attention
+
+  For more insights on Cilium's BGP, check out `eCHO Episode 101: More BGP fun with Cilium  <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
```
Comment (qmonnet) [inline Documentation/network/bgp-control-plane/bgp-control-plane.rst]: I don't remember the list of all episodes, but “more BGP fun” implies there's a former episode dealing with BGP as well? If that's the case, would it be worth linking the two episodes here?

File: Documentation/network/servicemesh/gateway-api/gateway-api.rst [hunk 5]
```diff
@@ -33,6 +33,11 @@ tests are passed.
 - `TLSRoute (experimental) <https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.TLSRoute>`__
 - `ReferenceGrant <https://gateway-api.sigs.k8s.io/api-types/referencegrant/>`_
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like more insights on Cilium's Gateway API support, check out `Nick Young's explanation in eCHO episode 58 <https://www.youtube.com/watch?v=60epwCxO8G4&index=80&t=2024s>`__.
```
Comment (qmonnet) [inline Documentation/network/servicemesh/gateway-api/gateway-api.rst]: I'd maybe keep the name of the presenter out of this, for fairness and consistency? This seems to be the only occurrence where you introduce the speaker?

File: Documentation/security/network/proxy/envoy.rst [hunk 6]
```diff
@@ -454,36 +459,36 @@ as an example:
...
-Note that we run both containers with labels that will make it easy to refer to these containers in a cilium 
-network policy.   Note that we have the client container run the sleep command, as we will use 'docker exec' to 
-access the client CLI.  
 
-Use ''cilium-dbg endpoint list'' to identify the IP address of the protocol server.  
+Note that we run both containers with labels that will make it easy to refer to these containers in a cilium
+network policy.   Note that we have the client container run the sleep command, as we will use 'docker exec' to
+access the client CLI.
+
+Use ''cilium-dbg endpoint list'' to identify the IP address of the protocol server.
```
Comment (qmonnet) [inline Documentation/security/network/proxy/envoy.rst]: If we changed that line we should also fix the quotes (should be back quotes instead of straight quotes).

**Review by qmonnet — CHANGES_REQUESTED** [review]
The addition of the video links look good to me overall, thanks! I've got a few comments that you can find inline below.

I have a concern with the rest of the changes, though. I suspect your text editor is automatically trimming spurious spaces at the end of the lines when you save your file. But I'd recommend against committing such massive change without first discussing it in #sig-docs, and without mentioning it in your commit title or description.

I'd recommend dropping these space clean-ups. I don't particularly like the unnecessary spaces, but they're not harmful, and the clean-up will complicate the Git history as well as any documentation backport. If we wanted to clean these up, this is something we should do on the whole documentation, in a separate PR (or at the very least in a separate commit).

File: Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst [hunk 7]
```diff
@@ -105,6 +105,11 @@ the mutual authentication feature:
 
    mutual-authentication-example
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like a video explanation and demo of Mutual Authentication in Cilium, check out `eCHO episode 100: Next-gen mutual authentication in Cilium  <https://www.youtube.com/watch?v=60epwCxO8G4&list=PLDg_GiBbAx-mY3VFLPbLHcxo6wUjejAOC&index=80&t=2024s>`__.
```
Comment (mhofstetter) [inline Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst]: Seems to be the wrong link (Gateway API)

File: Documentation/network/servicemesh/index.rst [hunk 8]
```diff
@@ -41,6 +41,11 @@ Looking at the feature set of a service mesh today, it can be summarized as foll
   transparent manner, i.e. without requiring to change application code.
 
 
+.. admonition:: Video
+  :class: attention
+
+  If you'd like a video explanation of Cilium's Service Mesh implementation, check out `eCHO episode 27: eBPF-enabled Service Mesh <https://www.youtube.com/watch?v=nJT0ASbGLvs>`__.
```
Comment (mhofstetter) [inline Documentation/network/servicemesh/index.rst]: Why not  link the newer episode 100 instead / in addition? https://www.youtube.com/watch?v=BWjDlynXhzg

edit: i realized that this is about service mesh in general - hence episode 27 is perfect

**Review by mhofstetter — CHANGES_REQUESTED** [review]
Thanks @hacktivist123! Some comments inline.

File: Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst (hunk 7 above)
Comment (hacktivist123) [reply Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst]: Ah yes! good catch, will fix.

File: Documentation/network/servicemesh/index.rst (hunk 8 above)
Comment (hacktivist123) [reply Documentation/network/servicemesh/index.rst]: i'll include this in addition rather than taking out the old one.

File: Documentation/installation/k8s-install-helm.rst (hunk 2 above)
Comment (hacktivist123) [reply Documentation/installation/k8s-install-helm.rst]: got it

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst (hunk 4 above)
Comment (hacktivist123) [reply Documentation/network/bgp-control-plane/bgp-control-plane.rst]: not really, there's no episode leading to this actually, It's just the title that might be confusing

File: Documentation/network/servicemesh/gateway-api/gateway-api.rst (hunk 5 above)
Comment (hacktivist123) [reply Documentation/network/servicemesh/gateway-api/gateway-api.rst]: fair enough, will fix

**Review by mhofstetter — APPROVED** [review]
`sig-servicemesh` :heavy_check_mark:

File: Documentation/security/network/proxy/envoy.rst (hunk 6 above)
Comment (hacktivist123) [reply Documentation/security/network/proxy/envoy.rst]: fixed

Comment (hacktivist123) [reply Documentation/security/network/proxy/envoy.rst]: I'll have to create a separate PR for this because the entire page has this issue about 23 occurrences

File: Documentation/installation/k8s-install-helm.rst [hunk 9]
```diff
@@ -95,7 +95,6 @@ Install Cilium
             --namespace kube-system \\
             --set aksbyocni.enabled=true \\
             --set nodeinit.enabled=true
-         
          .. note::
 
             Installing Cilium via helm is supported only for AKS BYOCNI cluster and
```
Comment (qmonnet) [inline Documentation/installation/k8s-install-helm.rst]: This one would be nice to fix, please. You moved the note out of the `parsed-literal` block in your first version, which was a good idea - just have it as a separate commit, please.

File: Documentation/installation/k8s-install-helm.rst [hunk 10]
```diff
@@ -156,7 +155,7 @@ Install Cilium
             2. Flush iptables rules added by VPC CNI
 
                .. code-block:: shell-session
-               
+
```
Comment (qmonnet) [inline Documentation/installation/k8s-install-helm.rst]: Unrelated to the PR.

File: Documentation/network/l2-announcements.rst [hunk 11]
```diff
@@ -46,7 +46,6 @@ The L2 Announcements feature and all the requirements can be enabled as follows:
                --set kubeProxyReplacement=true \\
                --set k8sServiceHost=${API_SERVER_IP} \\
                --set k8sServicePort=${API_SERVER_PORT}
-
```
Comment (qmonnet) [inline Documentation/network/l2-announcements.rst]: Unrelated to the PR.

File: Documentation/network/l2-announcements.rst [hunk 12]
```diff
@@ -107,13 +111,13 @@ where, and how. This is an example policy using all optional fields:
       interfaces:
       - ^eth[0-9]+
       externalIPs: true
-      loadBalancerIPs: true  
+      loadBalancerIPs: true
```
Comment (qmonnet) [inline Documentation/network/l2-announcements.rst]: Same, plus you even kept one space at the end of the line! :smile:

File: Documentation/network/l2-announcements.rst [hunk 13]
```diff
@@ -107,13 +111,13 @@ where, and how. This is an example policy using all optional fields:
...
       externalIPs: true
-      loadBalancerIPs: true  
+      loadBalancerIPs: true 
 
 Service Selector
 ----------------
 
 The service selector is a `label selector <https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/>`__ 
-that determines which services are selected by this policy. If no service 
+that determines which services are selected by this policy. If no service
```
Comment (qmonnet) [inline Documentation/network/l2-announcements.rst]: Unrelated

File: Documentation/network/l2-announcements.rst [hunk 14]
```diff
@@ -183,7 +187,7 @@ For example if an invalid match expression is provided:
 
   $ kubectl describe l2announcement 
   Name:         policy1
-  Namespace:    
+  Namespace:
```
Comment (qmonnet) [inline Documentation/network/l2-announcements.rst]: Remaining spaces, and it's unrelated anyway

File: Documentation/network/l2-announcements.rst [hunk 15]
```diff
@@ -374,7 +378,7 @@ When a node becomes the leader/holder, it will send out a gratuitous ARP reply
 over all of the configured interfaces. Clients who accept these will update 
 their ARP tables at once causing them to send traffic to the new leader/holder.
 Not all clients accept gratuitous ARP replies since they can be used for ARP spoofing. 
-Such clients might experience longer downtime then configured in the leases 
+Such clients might experience longer downtime then configured in the leases
```
Comment (qmonnet) [inline Documentation/network/l2-announcements.rst]: Unrelated

File: Documentation/network/servicemesh/gateway-api/gateway-api.rst [hunk 16]
```diff
@@ -58,4 +63,4 @@ More examples can be found in the `upstream repository <https://github.com/kuber
 Troubleshooting
 ###############
 
-.. include:: troubleshooting.rst
\ No newline at end of file
+.. include:: troubleshooting.rst
```
Comment (qmonnet) [inline Documentation/network/servicemesh/gateway-api/gateway-api.rst]: This one seems like an end-of-file line break fix, that's OK to keep.

File: Documentation/observability/grafana.rst [hunk 17]
```diff
@@ -16,6 +16,11 @@ Install Prometheus & Grafana
 This is an example deployment that includes Prometheus and Grafana in a single
 deployment.
 
+.. admonition:: Video
+ :class: attention
+
+  You can also see Cilium and Grafana in action on `eCHO Episode 68: Cilium & Grafana  <https://www.youtube.com/watch?v=DdWksYq5Pv4>`__.
```
Comment (qmonnet) [inline Documentation/observability/grafana.rst]:
```suggestion
  You can also see Cilium and Grafana in action on `eCHO episode 68: Cilium & Grafana <https://www.youtube.com/watch?v=DdWksYq5Pv4>`__.
```

File: Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst [hunk 18]
```diff
@@ -105,6 +105,11 @@ the mutual authentication feature:
 
    mutual-authentication-example
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like a video explanation and demo of Mutual Authentication in Cilium, check out `eCHO episode 100: Next-gen mutual authentication in Cilium  <https://www.youtube.com/watch?v=BWjDlynXhzg>`__.
```
Comment (qmonnet) [inline Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst]:
```suggestion
  If you'd like a video explanation and demo of Mutual Authentication in Cilium, check out `eCHO episode 100: Next-gen mutual authentication in Cilium <https://www.youtube.com/watch?v=BWjDlynXhzg>`__.
```

File: Documentation/network/l2-announcements.rst [hunk 19]
```diff
@@ -84,6 +83,11 @@ Limitations
 * The feature currently has no traffic balancing mechanism so nodes within the
   same policy might be asymmetrically loaded. For details see :ref:`l2_announcements_leader_election`.
 
+.. admonition:: Video
+  :class: attention
+
+  For more insights on Cilium's BGP, check out `eCHO Episode 101: More BGP fun with Cilium  <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
```
Comment (qmonnet) [inline Documentation/network/l2-announcements.rst]:
```suggestion
  For more insights on Cilium's BGP, check out `eCHO episode 101: More BGP fun with Cilium <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
```

File: Documentation/network/kubernetes/bandwidth-manager.rst [hunk 20]
```diff
@@ -260,4 +260,4 @@ Limitations
 .. admonition:: Video
   :class: attention
 
-  For more insights on Cilium's bandwidth manager, check out this `KubeCon talk on Better Bandwidth Management with eBPF <https://www.youtube.com/watch?v=QTSS6ktK8hY>`__.
+  For more insights on Cilium's bandwidth manager, check out this `KubeCon talk on Better Bandwidth Management with eBPF <https://www.youtube.com/watch?v=QTSS6ktK8hY>`__ and `eCHO Episode 98: Exploring the bandwidth manager with Cilium  <https://www.youtube.com/watch?v=-JnXe8vAUKQ>`__.
```
Comment (qmonnet) [inline Documentation/network/kubernetes/bandwidth-manager.rst]:
```suggestion
  For more insights on Cilium's bandwidth manager, check out this `KubeCon talk on Better Bandwidth Management with eBPF <https://www.youtube.com/watch?v=QTSS6ktK8hY>`__ and `eCHO episode 98: Exploring the bandwidth manager with Cilium <https://www.youtube.com/watch?v=-JnXe8vAUKQ>`__.
```

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst [hunk 21]
```diff
@@ -15,6 +15,10 @@ from outside the cluster for environments that support BGP. Because BGP
...
```
Comment (qmonnet) [inline Documentation/network/bgp-control-plane/bgp-control-plane.rst]:
```suggestion
  For more insights on Cilium's BGP, check out `eCHO episode 101: More BGP fun with Cilium <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
```

File: Documentation/installation/k8s-install-helm.rst [hunk 22]
```diff
@@ -221,6 +220,11 @@ Install Cilium
 
         .. include:: ../installation/alibabacloud-eni.rst
 
+.. admonition:: Video
+  :class: attention
+
+  If you'd like to learn more about Cilium Helm values, check out `eCHO episode 117: eCHO Episode 117: A Tour of the Cilium Helm Values <https://www.youtube.com/watch?v=ni0Uw4WLHYo>`__.
```
Comment (qmonnet) [inline Documentation/installation/k8s-install-helm.rst]:
```suggestion
  If you'd like to learn more about Cilium Helm values, check out `eCHO episode 117: A Tour of the Cilium Helm Values <https://www.youtube.com/watch?v=ni0Uw4WLHYo>`__.
```

File: Documentation/configuration/sctp.rst [hunk 23]
```diff
@@ -16,6 +16,11 @@ Enabling
 ========
 Pass ``--set sctp.enabled=true`` to helm.
 
+.. admonition:: Video
+ :class: attention
+
+  You can also watch a video explanation of Cilium's SCTP support in `eCHO Episode 78: Stream Control Transmission Protocol (SCTP) <https://www.youtube.com/watch?v=2lD86qNHXXI>`__.
```
Comment (qmonnet) [inline Documentation/configuration/sctp.rst]: Consistent case on “episode”, please.

```suggestion
  You can also watch a video explanation of Cilium's SCTP support in `eCHO episode 78: Stream Control Transmission Protocol (SCTP) <https://www.youtube.com/watch?v=2lD86qNHXXI>`__.
```

File: Documentation/network/egress-gateway.rst [hunk 24]
```diff
@@ -38,6 +38,11 @@ specific workload.
     information. If unsure, run ``cilium status`` and validate that Cilium is up
     and running.
 
+.. admonition:: Video
+  :class: attention
+
+  For more insights on Cilium's Egress Gateway, check out this `eCHO Episode 76: Cilium Egress Gateway <https://www.youtube.com/watch?v=zEQdgNGa7bg>`__.
```
Comment (qmonnet) [inline Documentation/network/egress-gateway.rst]:
```suggestion
  For more insights on Cilium's Egress Gateway, check out this `eCHO episode 76: Cilium Egress Gateway <https://www.youtube.com/watch?v=zEQdgNGa7bg>`__.
```

File: Documentation/network/kubernetes/intro.rst [hunk 25]
```diff
@@ -35,6 +35,11 @@ cluster:
   traffic.
 * Fully compatible with existing kube-proxy model
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like to learn more about Kubernetes networking and Cilium, check out `eCHO Episode 99: Explain Kubernetes Networking and Cilium to Network Engineers <https://www.youtube.com/watch?v=kMPKzb_kW68>`__.
```
Comment (qmonnet) [inline Documentation/network/kubernetes/intro.rst]:
```suggestion
  If you'd like to learn more about Kubernetes networking and Cilium, check out `eCHO episode 99: Explain Kubernetes Networking and Cilium to Network Engineers <https://www.youtube.com/watch?v=kMPKzb_kW68>`__.
```

File: Documentation/security/network/encryption.rst [hunk 26]
```diff
@@ -20,6 +20,11 @@ traffic between Cilium-managed endpoints either using IPsec or WireGuard®:
    encryption-ipsec
    encryption-wireguard
 
+.. admonition:: Video
+ :class: attention
+
+  You can also see a demo of Cilium Transparent Encryption in `eCHO Episode 79: Transparent Encryption with IPsec and WireGuard <https://www.youtube.com/watch?v=vj7M-t9MK6s>`__.
```
Comment (qmonnet) [inline Documentation/security/network/encryption.rst]:
```suggestion
  You can also see a demo of Cilium Transparent Encryption in `eCHO episode 79: Transparent Encryption with IPsec and WireGuard <https://www.youtube.com/watch?v=vj7M-t9MK6s>`__.
```

**Comment by qmonnet** [comment]
Checkpatch complains about your commit title being too long. I suspect this is because you may not have a blank line separator between your commit title and the commit description (consisting in your sign-off tag). The GitHub interface doesn't care about it, but Git usually does.

File: Documentation/installation/k8s-install-helm.rst (hunk 9 above)
Comment (hacktivist123) [reply Documentation/installation/k8s-install-helm.rst]: Okay!

File: Documentation/network/kubernetes/intro.rst [hunk 27]
```diff
@@ -79,4 +84,3 @@ Networking Model
 <https://kubernetes.io/docs/concepts/cluster-administration/networking/>`_ and
 `Kubernetes Network Plugins
 <https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/>`_.
-
```
Comment (qmonnet) [inline Documentation/network/kubernetes/intro.rst]: Unrelated.

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst [hunk 28]
```diff
@@ -78,4 +82,5 @@ Troubleshooting and Operation Guide
     :glob:
 
     bgp-control-plane-troubleshooting
-    bgp-control-plane-operation
\ No newline at end of file
+    bgp-control-plane-operation
+
```
Comment (qmonnet) [inline Documentation/network/bgp-control-plane/bgp-control-plane.rst]: Sorry I wasn't clear. I meant that your editor added a new line character at the end of the line, which was why we'd observe a diff here. I did not mean you should add an additional new empty line.

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst [hunk 29]
```diff
@@ -15,6 +15,10 @@ from outside the cluster for environments that support BGP. Because BGP
 Control Plane does not program the :ref:`datapath <ebpf_datapath>`, do not use it to establish
 reachability within the cluster.
 
+.. admonition:: Video
+  :class: attention
+
+  For more insights on Cilium's BGP, check out `eCHO Episode 101: More BGP fun with Cilium <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
 .. _Border Gateway Protocol: https://datatracker.ietf.org/doc/html/rfc4271
```
Comment (qmonnet) [inline Documentation/network/bgp-control-plane/bgp-control-plane.rst]: Please add an empty line here after the admonition.

Comment (qmonnet) [reply Documentation/network/bgp-control-plane/bgp-control-plane.rst]: Also, s/Episode/episode/.

File: Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst [hunk 30]
```diff
@@ -20,13 +20,13 @@ Mutual Authentication (Beta)
 Mutual Authentication and mTLS Background
 #########################################
 
-Mutual Transport Layer Security (mTLS) is a mechanism that ensures the authenticity, integrity, 
+Mutual Transport Layer Security (mTLS) is a mechanism that ensures the authenticity, integrity,
```
Comment (qmonnet) [inline Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst]: This file still contains the space changes.

**Review by qmonnet — CHANGES_REQUESTED** [review]
Requesting changes, as per the comments above

File: Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst (hunk 30 above)
Comment (hacktivist123) [reply Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst]: I wonder how this file escaped my settings, I made sure to double check

Comment (hacktivist123) [reply Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst]: Will fix

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst (hunk 29 above)
Comment (hacktivist123) [reply Documentation/network/bgp-control-plane/bgp-control-plane.rst]: Got it

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst (hunk 28 above)
Comment (hacktivist123) [reply Documentation/network/bgp-control-plane/bgp-control-plane.rst]: Oh alright!

File: Documentation/network/kubernetes/intro.rst (hunk 27 above)
Comment (hacktivist123) [reply Documentation/network/kubernetes/intro.rst]: This trailing space is actually in the file already, if you edit the file, you'd see its there. I'm not sure why Github isn't picking it up

File: Documentation/installation/cni-chaining-azure-cni.rst [hunk 31]
```diff
@@ -21,6 +21,12 @@ Azure CNI (Legacy)
 
 .. include:: cni-chaining-limitations.rst
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like a video explanation of the Azure CNI Powered by Cilium, check out `eCHO episode 70: Azure CNI Powered by Cilium <https://www.youtube.com/live/8it8Hm2F_GM?si=E_z57MLGJZeZJSyN>`__.
```
Comment (qmonnet) [inline Documentation/installation/cni-chaining-azure-cni.rst]: I'm also noting that this link doesn't have the same format as the others (`.../live/...` instead of `.../watch?v=...`), is this on purpose?

Comment (hacktivist123) [reply Documentation/installation/cni-chaining-azure-cni.rst]: Not really, its a youtube thing for live videos, it depends on how you access the video. If it's from the playlist then it'll use the latter format but if its from the live streams tab, it'll use the former. However, I've updated the link to use the latter format

**Review by youngnick — APPROVED** [review]
LGTM from sig-k8s

File: Documentation/network/kubernetes/intro.rst (hunk 27 above)
Comment (qmonnet) [reply Documentation/network/kubernetes/intro.rst]: There's no trailing space here. This is a blank line that you remove in your PR.

File: Documentation/installation/cni-chaining-azure-cni.rst [hunk 32]
```diff
@@ -24,7 +24,7 @@ Azure CNI (Legacy)
 .. admonition:: Video
  :class: attention
 
-  If you'd like a video explanation of the Azure CNI Powered by Cilium, check out `eCHO episode 70: Azure CNI Powered by Cilium <https://www.youtube.com/live/8it8Hm2F_GM?si=E_z57MLGJZeZJSyN>`__.
+  If you'd like a video explanation of the Azure CNI Powered by Cilium, check out `eCHO episode 70: Azure CNI Powered by Cilium <https://www.youtube.com/watch?v=8it8Hm2F_GM>`__.
```
Comment (qmonnet) [inline Documentation/installation/cni-chaining-azure-cni.rst]: This fix should be part of the first commit, not the second, please.

File: Documentation/installation/k8s-install-helm.rst [hunk 33]
```diff
@@ -86,21 +86,20 @@ Install Cilium
        .. include:: ../installation/requirements-aks.rst
 
        **Install Cilium:**
-
        Deploy Cilium release via Helm:
```
Comment (qmonnet) [inline Documentation/installation/k8s-install-helm.rst]: Unrelated, and undesirable. I didn't check the preview but I expect that removing this line will make Sphinx concatenate both line as a single paragraph, when the first one is meant to be some sort of sub-heading.

File: Documentation/installation/k8s-install-helm.rst [hunk 34]
```diff
@@ -86,21 +86,20 @@ Install Cilium
...
 
+       .. note::
+
+            Installing Cilium via helm is supported only for AKS BYOCNI cluster and
+            not for Azure CNI Powered by Cilium clusters.
+
```
Comment (qmonnet) [inline Documentation/installation/k8s-install-helm.rst]: Looking again at this change, can we keep the note _below_ the command line? It's strange to introduce the command with “Deploy Cilium release via Helm:” but then to have a note between this and the actual command.

The fix would simply consist in un-indenting the note block. so that `.. note::` is indented at the same level as `.. parsed-literal::`.

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst [hunk 35]
```diff
@@ -18,7 +18,8 @@ reachability within the cluster.
 .. admonition:: Video
   :class: attention
 
-  For more insights on Cilium's BGP, check out `eCHO Episode 101: More BGP fun with Cilium <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
+  For more insights on Cilium's BGP, check out `eCHO episode 101: More BGP fun with Cilium <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
+
```
Comment (qmonnet) [inline Documentation/network/bgp-control-plane/bgp-control-plane.rst]: This fix is part of the second commit, but should be moved to the first commit, please.

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst [hunk 36]
```diff
@@ -82,5 +83,4 @@ Troubleshooting and Operation Guide
     :glob:
 
     bgp-control-plane-troubleshooting
-    bgp-control-plane-operation
-
+    bgp-control-plane-operation
```
Comment (qmonnet) [inline Documentation/network/bgp-control-plane/bgp-control-plane.rst]: This change in the second commit is to cancel the change from the first commit. Please fix in the first commit, not in the second!

File: Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst [hunk 37]
```diff
@@ -20,13 +20,13 @@ Mutual Authentication (Beta)
...
-Mutual Transport Layer Security (mTLS) is a mechanism that ensures the authenticity, integrity,
+Mutual Transport Layer Security (mTLS) is a mechanism that ensures the authenticity, integrity, 
 and confidentiality of data exchanged between two entities over a network.
 
-Unlike traditional TLS, which involves a one-way authentication process where the client verifies the
+Unlike traditional TLS, which involves a one-way authentication process where the client verifies the 
 server's identity, mutual TLS adds an additional layer of security by requiring both the client and the server to authenticate each other.
 
-Mutual TLS aims at providing authentication, confidentiality and integrity to service-to-service communications.
+Mutual TLS aims at providing authentication, confidentiality and integrity to service-to-service communications.
```
Comment (qmonnet) [inline Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst]: All the whitespace fixes in Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst are still present in your PR: You remove the spaces in the first commit, and you re-add them in the second commit. Please discard these changes altogether.

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst (hunk 35 above)
Comment (qmonnet) [inline Documentation/network/bgp-control-plane/bgp-control-plane.rst]: Git says you have trailing spaces on this line that you had, please let's not introduce new trailing spaces if possible :)

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thanks!

Please make sure to review your commits individually. The second contains fixes that should be in the first commit.

File: Documentation/installation/k8s-install-helm.rst (hunk 34 above)
Comment (hacktivist123) [reply Documentation/installation/k8s-install-helm.rst]: alright

File: Documentation/network/bgp-control-plane/bgp-control-plane.rst (hunk 35 above)
Comment (hacktivist123) [reply Documentation/network/bgp-control-plane/bgp-control-plane.rst]: fixed

File: Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst (hunk 37 above)
Comment (hacktivist123) [reply Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst]: fixed

**Review by qmonnet — APPROVED** [review]
I addressed the conflict caused by the merge of the BGP docs update.

This version looks good to me, thanks a lot for this work and for bearing with me and my nits! :slightly_smiling_face:

**Comment by hacktivist123** [comment]
Thanks for all the help! @qmonnet you rock! 🚀

**Comment by qmonnet** [comment]
This is not a technical documentation update, so the reviews from the different code owners can be skipped. Besides, we already have three positive reviews for the PR. I hereby decide we're good to go :ship:

**Review by harsimran-pabla — APPROVED** [review]
BGP change lgtm
